app/src/main/res/layout/fragment_product_detail.xml

```diff
@@ -262,35 +285,36 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_alignParentBottom="true"
-                android:background="@color/color_white"
-                android:orientation="horizontal"
-                android:padding="@dimen/dp_3"
+                android:background="@color/color_gray_white"
+                tools:ignore="UselessParent"
                 >
-                <ImageView
-                    android:id="@+id/image_user"
-                    android:layout_width="@dimen/dp_0"
-                    android:layout_height="@dimen/dp_45"
-                    android:layout_marginRight="@dimen/dp_3"
-                    android:layout_weight="1.3"
-                    android:padding="@dimen/dp_1"
-                    app:imageUrl="@{viewModel.userImage}"
-                    />
-                <EditText
-                    android:layout_width="@dimen/dp_0"
+                <LinearLayout
+                    android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:layout_weight="7"
-                    android:hint="@string/add_a_comment"
-                    android:text="@={viewModel.comment}"
-                    />
-                <ImageButton
-                    android:layout_width="@dimen/dp_0"
-                    android:layout_height="@dimen/dp_35"
-                    android:layout_marginLeft="@dimen/dp_3"
-                    android:layout_weight="1"
-                    android:background="@drawable/ic_send_gray"
-                    android:onClick="@{() -> viewModel.onClickSendComment()}"
-                    />
-
+                    android:layout_margin="@dimen/dp_7"
+                    android:background="@drawable/border_comment"
+                    android:orientation="horizontal"
+                    android:padding="@dimen/dp_3"
+                    android:contentDescription="@string/add_a_comment"
+                    >
+                    <EditText
+                        android:layout_width="@dimen/dp_0"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="@dimen/dp_3"
```

Comment:
ERROR: [RtlCompat] To support older versions than API 17 (project specifies 14) you should *also* add `android:layout_marginLeft="@dimen/dp_3"`
 API 17 adds a `textAlignment` attribute to specify text alignment. However, if you are supporting older versions than API 17, you must *also* specify a gravity or layout_gravity attribute, since older platforms will ignore the `textAlignment` attribute.